### PR TITLE
chat controls upgrade

### PR DESCRIPTION
**Pull request by CDWimmer**
chatControls are now self-consistent and all images, no more emoji :)

### CODE REVIEW

File: desktop/assets/css/desktop.css [hunk 1]
```diff
@@ -317,7 +317,7 @@ div.ui-resizable-handle {
   */
   margin: 0 0 0 0px;
   padding: 0 0 0 0px;
-  min-height: 77%;
+  /* min-height: 77%; NO!! */
```
Comment (Marak) [inline desktop/assets/css/desktop.css:320]: @CDWimmer - Everything is looks GOOD except without this line the chat window doesnt fill unless it hits enough messages.

Makes the default state of fresh chat window look odd ( screenshot attached )


Screen Shot 2022-04-11 at 6 18 23 PM

**Comment by Marak** [comment]
@CDWimmer -

Thank you for your contributions.

We'd like to merge this but the history is off. The PR should be from clean master branch not your branch with master being merged in.

If you can reopen the PR with clean history we can merge. Thanks.

File: desktop/assets/css/desktop.css (hunk 1 above)
Comment (CDWimmer) [reply desktop/assets/css/desktop.css:320]: Dang it, thought I'd fixed that, must've not transferred to buddy chat.

**Comment by CDWimmer** [comment]
> 

Not quite sure what you mean nor how to fix it :( I'm lacking familiarity with git

**Pull request closed without merging**